nb_lib/formatter.py

```diff
@@ -23,15 +29,31 @@ def __init__(self, output):
         else:
             self.exporter = MarkdownExporter()
 
+    def get_output_dir_base(self, file):
```

Comment:
this will probably change when merged with upstream https://github.com/fernandezpablo85/notebook_convert/commit/9b46aefcaa62fc4567747f5d9fc178df51d83f89